drivers/sensor/lm75/Kconfig

```
@@ -10,4 +10,43 @@ config LM75
 	select I2C
 	help
 	  Enable the driver for the LM75 digital temperature Sensor with 2-wire
-	  interface. Thermal Watchdog is not implemented
+	  interface.
+
+if LM75
+
+choice
+	prompt "Trigger mode"
+	default LM75_TRIGGER_NONE
+	help
+	  Specify the type of triggering to be used by the driver.
+
+config LM75_TRIGGER_NONE
+	bool "No trigger"
+
+config LM75_TRIGGER_GLOBAL_THREAD
+	bool "Use global thread"
+	depends on GPIO
+	select LM75_TRIGGER
```

Not sure on the exact syntax, but the idea is that we don't enable trigger support unless we have `int-gpios` defined.
```suggestion
config LM75_TRIGGER_GLOBAL_THREAD
	bool "Use global thread"
	depends on $(dt_compat_any_has_prop,$(DT_COMPAT_LM75),int-gpios))
	select LM75_TRIGGER
```

---

#74342